src/test/java/uk/gov/ons/ctp/response/collection/exercise/endpoint/CollectionExerciseClient.java

```diff
@@ -280,4 +280,31 @@ public void createCollectionExerciseEvent(EventDTO event) {
               event.getCollectionExerciseId(), event.getTag(), response.getStatus()));
     }
   }
+
+  void sampleSummaryReadiness(final UUID sampleSummaryId) throws CTPException {
+    sampleSummaryReady(sampleSummaryId);
+  }
+
+  private int sampleSummaryReady(final UUID sampleSummaryId) throws CTPException {
+    try {
+      JSONObject jsonPayload = new JSONObject();
+      JSONArray jsonSampleSummaryId = new JSONArray();
```

Comment:
vulnerability on JSONArray
https://devhub.checkmarx.com/cve-details/Cx08fcacc9-cb99/?utm_source=jetbrains&utm_medium=referral&utm_campaign=idea

Reply:
Out of scope for this card